src/components/progress-steps/progress-steps.jsx

```diff
@@ -93,6 +95,7 @@ const ProgressStep = ( {
 	size,
 	isLast,
 	index,
+	lineProps,
```

Comment:
```suggestion
	lineProps = 'min-w-10',
```
These can be set as default.

Reply:
this is already set as default in parent component, on line 17